runestone/common/project_template/_templates/plugin_layouts/sphinx_bootstrap/layout.html

```diff
@@ -11,16 +11,11 @@
     '_static'+'/processing-1.4.1.min.js',
     '_static'+'/jquery.hotkey.js',
     '_static'+'/jquery-migrate-1.2.1.min.js',
-    '_static'+'/presenter_mode.js',    
+    '_static'+'/presenter_mode.js',
   ]
 %}
 
 {% set css_files = ['_static/bootstrap-3.0.0/css/bootstrap.min.css', '_static/presenter_mode.css'] + css_files %}
-{% if theme_bootswatch_theme %}
```

Comment:
This does not break anything??!!

Reply:
Just the opposite, it fixes something. Before this patch, I saw the line:

`    <link rel="stylesheet" href="http://netdna.bootstrapcdn.com/bootswatch/2.3.1/""/bootstrap.min.css" type="text/css" />
`

in all the output HTML files. The deletion in this PR removes these (broken) lines from the output.

Reply:
I think that might be because of an issue in `theme.conf`, not a problem here.  I ran into an issue where if the `bootswatch_theme` variable is set to `""` (as is done in the included `theme.conf` and as [the comment above](https://github.com/RunestoneInteractive/RunestoneComponents/blob/fa19c4bdff33649145d6683c507338c56f7499b7/runestone/common/project_template/_templates/plugin_layouts/sphinx_bootstrap/theme.conf#L42) suggests is the default), then it includes the `""` you're seeing.  I believe the line in `theme.conf` should be `bootswatch_theme = `, without any quotes, to use no theme.  Just based on my recollection, anyway.

Reply:
Well, I should probably add to that:  The template pulls in Bootstrap locally from the static/ folder somewhere else, hence why it doesn't cause problems when the CDN bootswatch CSS fails to load.  If you correct the `bootswatch_theme` setting, then pages will pull in Bootstrap twice (and I think with two different versions).  So it's probably best to simplify by either dropping the bootswatch theme loading altogether and just use the local Bootstrap, or to move entirely to CDN bootstrap and drop the locally-hosted version.

Reply:
@liffiton, thanks for the feedback. IMHO, if Runestone should be capable of working locally (without a server), then Bootstrap should be stored locally. So, I'll amend this PR by removing the incorrect settings in `theme.conf`, as well as the removal already done in this PR.